src/zarr/core/group.py

```diff
@@ -101,7 +150,17 @@ def to_buffer_dict(self, prototype: BufferPrototype) -> dict[str, Buffer]:
                 ),
             }
 
-    def __init__(self, attributes: dict[str, Any] | None = None, zarr_format: ZarrFormat = 3):
+    # def __post_init__(self):
```

Comment:
This is leftover from testing and can be removed. FWIW, `__post_init__` seemed to work fine.